packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -345,9 +352,14 @@ contract OPStackManager is ISemver, Initializable {
         return Blueprint.deployFrom(blueprint.proxy, salt, abi.encode(_proxyAdmin));
     }
 
-    /// @notice Returns the implementation data for a contract name.
-    function getLatestImplementation(string memory _name) internal view returns (Implementation storage) {
-        return implementations[latestRelease][_name];
+    /// @notice Returns the implementation data for a contract name. Makes a copy of the internal
```

Comment:
@maurelian @mds1 - I updated this method to return a copy of the underlying data in storage. The code [here](https://github.com/ethereum-optimism/optimism/pull/11955/files#diff-adc000b158932be783e0ea3e4fac762b4aa8a6310d4088675627630dc977eaddR302-R306) was overwriting the DisputeGameFactory's release info in the `implementations` map, which was causing subsequent deployments using the OPSM to fail.

Reply:
Nice catch! That makes sense. This fix is ok to unblock this PR, in a future PR I'd like to change back from `memory` to `storage` (it will be cheaper and smaller code), then we just don't assign to `impl` for the ASR logic there